websites/D/Dash Radio/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "apprehends",
+    "id": "202236922217758721"
+  },
+  "service": "Dash Radio",
+  "description": {
+    "en": "Dash Radio is a premium radio website without the ads. Available on over 50 platforms. (IN ORDER FOR THIS TO WORK, TEXT MUST BE STATIC NOT SCROLLING)"
+  },
+  "url": "dashradio.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/o4Ea2f9.png",
+  "thumbnail": "https://i.imgur.com/o4Ea2f9.png",
+  "color": "#55effa",
```

Comment:
thumbnail should be a promotional banner or a screenshot of the website homepage.